tests/baselines/reference/jsxInvalidEsprimaTestSuite.errors.txt

```diff
@@ -33,9 +33,7 @@ tests/cases/conformance/jsx/16.tsx(1,2): error TS17008: JSX element 'a' has no c
 tests/cases/conformance/jsx/16.tsx(1,10): error TS1005: '</' expected.
 tests/cases/conformance/jsx/17.tsx(1,2): error TS17008: JSX element 'a' has no corresponding closing tag.
 tests/cases/conformance/jsx/17.tsx(1,10): error TS1005: '</' expected.
-tests/cases/conformance/jsx/18.tsx(1,9): error TS2695: Left side of comma operator is unused and has no side effects.

```

Comment:
this was actually a good error usually... caught some obvious bugs.. do not know about disabling it by default..